### PR TITLE
Utilize businesses fetched from datastore

### PR DESCRIPTION
**Pull request by eash130**

### CODE REVIEW

File: capstone/src/main/webapp/js/business.js [hunk 1]
```diff
@@ -14,4 +14,20 @@
 
 import { loadCommentSection } from '/js/comments.js';
 
-window.onload = () => loadCommentSection(document.getElementById('comment-section'));
+window.onload = function() {
+  const url = new URLSearchParams(window.location.search);
```
Comment (lukasberglund) [inline capstone/src/main/webapp/js/business.js:18]: What does this do?

Comment (eash130) [reply capstone/src/main/webapp/js/business.js:18]: Could be helpful to add a comment here.

File: capstone/src/main/webapp/js/business.js [hunk 2]
```diff
@@ -14,4 +14,22 @@
 
 import { loadCommentSection } from '/js/comments.js';
 
-window.onload = () => loadCommentSection(document.getElementById('comment-section'));
+window.onload = function() {
+  // window.location.search returns the search string of the URL.
+  // In this case, window.location.search = ?id={businessID}
+  const url = new URLSearchParams(window.location.search);
```
Comment (lukasberglund) [inline capstone/src/main/webapp/js/business.js:20]: Why is this `const` but `businessId` is declared with `let`?

**Comment by lukasberglund** [comment]
This is great. If you could just resolve my one question that would be aweomse.

File: capstone/src/main/webapp/js/business.js (hunk 2 above)
Comment (eash130) [reply capstone/src/main/webapp/js/business.js:20]: no reason. I might as well use two const.

**Pull request merged**